ersilia/hub/fetch/fetch.py

```diff
@@ -258,8 +258,8 @@ async def fetch(self, model_id):
         except StandardModelExampleError:
             self.logger.debug("Standard model example failed, deleting artifacts")
             do_delete = yes_no_input(
-                "Do you want to delete the model artifacts? [Y/n]", default_answer="Y"
-            )
+                "Do you want to delete the model artifacts? [Y/n]", 
+                default_answer="n")
```

Comment:
@Abellegese any reason you changed this? The default answer should be Y for most users (who will not end up being developers). We need the model artifacts to be deleted so it doesn't get picked up via catalog command and such. This is essential for user experience bec otherwise users would be able to serve it, and it may happen that the model doesn't actually work for them.

Reply:
Hi @DhanshreeA so I made this change because when I test `eos2db3` it fails to produce standard example files, since we can not use prompt while we are running the second `subprocess`(`fetch`) it automatically removes the artifact(assuming the default prompt is `yes`) and we cant run the test because its removed.

Reply:
I guess it removed the artifact because it fails to produce standard examples(total fetch status was `fetch failed`). It was not the case for other model.

Reply:
In that case @Abellegese you can set an environment variable for the shell running that subprocess like so: https://stackoverflow.com/a/4453495/1887515